recipes/libressl/2.9.2/test_package/conanfile.py

```diff
@@ -0,0 +1,24 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class PackageTest(ConanFile):
+    settings = "os", "compiler", "arch", "build_type"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if tools.cross_building(self.settings):
+            # Handle mingw
+            if self.settings.os == "Windows":
```

Comment:
it's not required, but it doesn't harm.

Reply:
It did break on my machine, because I use git-bash